sdcm/nemesis.py

```diff
@@ -250,14 +242,27 @@ def repair_nodetool_rebuild(self):
         self._run_nodetool(rebuild_cmd, self.target_node)
 
 
-def log_time_elapsed(method):
+def log_time_elapsed_and_status(method):
     """
     Log time elapsed for method to run
 
     :param method: Remote method to wrap.
     :return: Wrapped method.
     """
+    def print_nodetool_status(self):
+        for node in self.cluster.nodes:
```

Comment:
@amoskong 
do you mean to check the output of 'nodetool status'( the number of nodes before  and after nemesis)?
or check len(self.cluster.nodes) before and after?